compute_sdk/tests/unit/test_diagnostic.py

```diff
@@ -329,7 +329,8 @@ def test_diagnostic_gzip(
         assert len(captured_stdout) == 2
 
     assert captured_stdout[-2].startswith(
-        "Compressed diagnostic output successfully writen to globus_compute_diagnostic_"
+        "Compressed diagnostic output successfully written "
+        "to globus_compute_diagnostic_"
```

Comment:
Good deal; we've moved to make multi-line strings like this put the space at the head of the next line for easier visibility.  Not going to block this PR over it, just bringing it up:
```diff
- "Compressed diagnostic output successfully written "
- "to globus_compute_diagnostic_"
+ "Compressed diagnostic output successfully written"
+ " to globus_compute_diagnostic_"
```

Reply:
I used to prepend subsequent lines with spaces because I think it's much clearer to read. However, I think that the Automate team standardized itself otherwise.

Anyway: feedback incorporated!